uint64 timestamp 		# time since system start (microseconds)
uint8 is_available # mode
float32 temperature
float32 humidity
float32 pressure
float32 date1
float32 date2
